sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -4634,7 +4645,10 @@
                           "addons": [],
                           "id": "esc41a",
                           "inputType": "number",
-                          "tags": []
+                          "tags": [
+                            "execute-calculation",

```

Comment:
Is it not `execute-validation` enough here?
Please ensure that create and edit are behaving correctly. If I am not wrong during the creation there is a different behavior where the validations are delayed until the user pressed the validate or submit button.